README.md

```diff
@@ -114,7 +59,7 @@ For more information on the importance of a professional README for your reposit
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- [Live Demo Link]() 
```

Comment:
- [ ] The previous reviewer suggested that you remove the section completely since you do not have a deployed link.

  ![image](https://github.com/eudondian/opp-school-library/assets/36542986/3dab3ff9-2e39-460f-8ab0-cee94f6e8ec8)

- [ ] **[OPTIONAL]** Kindly also remove all section descriptions as highlighted by the previous reviewer like "Add a link to your deployed project." in the live demo section, and "Describe how a new developer could make use of your project." in the Getting Started section.